pytest/tests/sanity/routing_table_sync.py

```diff
@@ -0,0 +1,160 @@
+import sys, time
+
+import base58
+import ed25519
+
+sys.path.append('lib')
+
+from cluster import start_cluster
+from peer import logger
+from proxy import ProxyHandler
+from multiprocessing import Value
+
+TIMEOUT = 300
+success = Value('i', 0)
+
+
+success = Value('i', 0)
+
+class Handler(ProxyHandler):
+    async def handle(self, msg, fr, to):
+        if msg.enum == "RoutingTableSyncV2" and msg.RoutingTableSyncV2.enum == "Version2":
+            if msg.RoutingTableSyncV2.Version2.routing_state.enum == "Done":
+                success.value = 1
+            logger.info("ROUTING_STATE %s" % msg.RoutingTableSyncV2.Version2.routing_state.enum)
+            logger.info("* known_edges %s" % msg.RoutingTableSyncV2.Version2.known_edges)
+            logger.info("* edges %s" % len(msg.RoutingTableSyncV2.Version2.edges))
+        return True
+
+
+nodes = start_cluster(2, 0, 1, None, [], {}, Handler)
+
+time.sleep(10)
+
+signing_key, verifying_key = ed25519.create_keypair()
+
+logger.info(str(signing_key))
+
+
+def gen():
+    return base58.b58encode(ed25519.create_keypair()[0].to_bytes()).decode('ascii')
+
+
+def new_edge(peer_a, peer_b, nonce):
+    return {
+        'nonce': nonce,
+        'peer0': 'ed25519:' + base58.b58encode(str(peer_a).zfill(32)).decode("ascii"),
+        'peer1': 'ed25519:' + base58.b58encode(str(peer_b).zfill(32)).decode("ascii"),
+        'removal_info': None,
+        'signature0': 'ed25519:5ca6A3uzrK3FKGjeAEi7pLNDiBbrzQ1ZPseXF1Rw8rfr3RWYNXPckVDUGGKMZQFE8eKWXeeCws6ZvmeQW8PZEzP6',
+        'signature1': 'ed25519:5ca6A3uzrK3FKGjeAEi7pLNDiBbrzQ1ZPseXF1Rw8rfr3RWYNXPckVDUGGKMZQFE8eKWXeeCws6ZvmeQW8PZEzP6'
+    }
+
+
+def chunks(ary, n):
+    n = max(1, n)
+    return (ary[i:i + n] for i in range(0, len(ary), n))
+
+
+tests = [
+    [50000, 50000, 0, 10],
+    [3, 0, 0, 1],
+    [1, 11, 0, 1],
+#    [1000000, 1000000, 0],
```

Comment:
why is this disabled?

Reply:
This test is no longer needed. The first case handles an edge case where symmetrical difference is 100k, which exceeds the number of edges we can recover using IBF, which according to my estimates is around 76k.  Having a bigger test isn't beneficial beneficial, and it does take a while to run.